README.md

```diff
@@ -372,6 +373,20 @@ Available `options`:
 * `ttr` - time to release in seconds. The time after which, if there is no active
 connection in the session, it will be released with all its tasks.
 
+## Session identify
+
```

Comment:
I would highlight the ability we want to provide with this function somewhere at the start of the section, For example:

> Sometimes we need an ability to acknowledge a task after reconnect (because retrying it is undesirable) or even acknowlegde using another connection.

Reply:
Accepted.